### PR TITLE
 [MINOR] doc: Add JDK benchmark tests

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?
Add JDK benchmark tests

### Why are the changes needed?
We should let more users know to use JDK 11+ to deploy shuffle server.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Just doc.

### CODE REVIEW

**Comment by zuston** [comment]
Overall LGTM. Left minor comments

File: README.md [hunk 1]
```diff
@@ -177,7 +177,18 @@ rss-xxx.tgz will be generated for deployment
    ```
 
 ### Deploy Shuffle Server
-
+We recommend to use JDK 11+ if we want to have better performance when we deploy the shuffle server.
```
Comment (zuston) [inline README.md:180]: Indicate using the GRPC?

Comment (jerqi) [reply README.md:180]: done.

**Review by zuston — APPROVED** [review]
LGTM.

**Pull request merged**